src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/ParseJson_Arithmetic.txt

```diff
@@ -0,0 +1,259 @@
+﻿// Test simple arithmetic
+>> 1+ParseJSON("2")
+3
+
+>> Blank()+ParseJSON("2")
+2
+
+>> ParseJSON("1")+2
+3
+
+>> ParseJSON("1")+Blank()
+1
+
+>> ParseJSON("1")+ParseJSON("2")
+3
+
+>> 1 + (1/ParseJSON("0"))
+Error({Kind:ErrorKind.Div0})
+
+>> 5-ParseJSON("3")
+2
+
+>> Blank()-ParseJSON("3")
+-3
+
+>> ParseJSON("5")-3
+2
+
+>> ParseJSON("5")-Blank()
+5
+
+>> ParseJSON("5")-ParseJSON("3")
+2
+
+// Unary minus
+>> -ParseJSON("3")
+-3
+
+>> -(1/ParseJSON("0"))
+Error({Kind:ErrorKind.Div0})
+
+>> 5 - -ParseJSON("3")
+8
+
+>> -(5+ParseJSON("3"))
+-8
+
+>> -(ParseJSON("5")+3)
+-8
+
+>> 2*ParseJSON("3")
+6
+
+>> Blank()*ParseJSON("3")
+0
+
+>> ParseJSON("2")*3
+6
+
+>> ParseJSON("2")*Blank()
+0
+
+>> ParseJSON("2")*ParseJSON("3")
+6
+
+>> ParseJSON("15")/3
+5
+
+>> ParseJSON("15")/Blank()
+Error({Kind:ErrorKind.Div0})
+
+>> 15/ParseJSON("3")
+5
+
+>> Blank()/ParseJSON("3")
+0
+
+>> ParseJSON("15")/ParseJSON("3")
+5
+
+>> ParseJSON("1")/2
+0.5
+
+>> 1/ParseJSON("2")
+0.5
+
+>> ParseJSON("20")%
+0.2
+
+>> ParseJSON("200")%
+2
+
+>> (1/ParseJSON("0"))%
+Error({Kind:ErrorKind.Div0})
+
+>> ParseJSON("1") * 50%
+0.5
+
+>> 1 * ParseJSON("50")%
+0.5
+
+// Precedence
+>> ParseJSON("1")+2*3
+7
+
+>> 1+ParseJSON("2")*3
+7
+
+>> 1+2*ParseJSON("3")
+7
+
+>> ParseJSON("1")+ParseJSON("2")*ParseJSON("3")
+7
+
+// Comparison operations 
+>> ParseJSON("1")=1
+true
+
+>> 1=ParseJSON("1")
+true
+
+>> ParseJSON("1")<>1
+false
+
+>> 1<>ParseJSON("1")
+false
+
+>> ParseJSON("1")=Blank()
```

Comment:
>ParseJSON("1")=Blank() [](http://example.com/codeflow?start=3&length=22)

Some additional tests (sorry if they are defined further down):

    >> ParseJSON("0") = Blank()
    false

    >> ParseJSON("null") = Blank()
    true

Same with the order swapped

Reply:
I added these tests. It required adding a new operator implementation specifically for blank+untyped for the eq and neq operators.